health-services/project/src/main/java/org/egov/project/repository/LocationCaptureRepository.java

```diff
@@ -0,0 +1,133 @@
+package org.egov.project.repository;
```

Comment:
We don't need two repositories anymore. use the same repository, whatever feedback added here, check for same issues in user action repository also

Reply:
if making this into one repository will add unnecessary complex then it's fine but call out the reason

Reply:
During the filtering process of the find methods, which are explicitly defined for each entity type, issues may arise in later iterations of the code for UserAction. Specifically, when different search fields are defined for UserAction that are not relevant to LocationCapture, these fields might be included in the final constructed query. To avoid this, we may need to add multiple checks to determine whether the object is for LocationCapture, which adds an extra step. Further discussion on this is pending. However, using the same repository for both entities would be a better design approach and needs further optimization.